packages/dashboard/src/components/confirm-alert-dialog.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Slide from '@material-ui/core/Slide';
+import { TransitionProps } from '@material-ui/core/transitions';
+import WarningIcon from '@material-ui/icons/Warning';
+import { makeStyles } from '@material-ui/core';
+
+const Transition = React.forwardRef(function Transition(
+  props: TransitionProps & { children?: React.ReactElement<any, any> },
+  ref: React.Ref<unknown>,
+) {
+  return <Slide direction="up" ref={ref} {...props} />;
+});
+
+export interface ConfirmAlertDialogProps {
+  open: boolean; // controls when the modal should open
+  close: () => void; // close the modal
```

Comment:
The material ui pattern is to let the parent component handle closing the dialog from the confirm/cancel callback.

Reply:
Ok, let's make the parent handle the closing from the confirm/cancel.